graphql-dgs-codegen-shared-core/src/main/kotlin/com/netflix/graphql/dgs/client/codegen/GraphQLQueryRequest.kt

```diff
@@ -63,7 +64,11 @@ class GraphQLQueryRequest @JvmOverloads constructor(
         if (query.input.isNotEmpty()) {
```

Comment:
There's also the GraphQLQueryRequestTest for testing serialized queries with variables.